frontend/src/pages/Dashboard.jsx

```diff
@@ -0,0 +1,452 @@
+                    <div className="charts">
+                        <div className="usage-chart">
+                            <div className="div-2">
+                                <div className="text-wrapper">License Usage</div>
+                                <div className="div-4" />
+                            </div>
+                            <div style={{ marginTop: '120px' }}>
+                                <LineChart
+                                    width={850}
+                                    height={300}
+                                    data={LU_data}
+                                    margin={{
+                                        top: 5,
+                                        right: 40,
+                                        left: 30,
+                                        bottom: 5,
+                                    }}
+                                    >
+                                    <CartesianGrid strokeDasharray="3 3" />
+                                    <XAxis dataKey="name" />
+                                    <YAxis />
+                                    <Tooltip />
+                                    <Legend />
+                                    <Line type="monotone" dataKey="high" stroke="#FF0000" activeDot={{ r: 5 }} />
+                                    <Line type="monotone" dataKey="medium" stroke="#F4D600" />
+                                    <Line type="monotone" dataKey="low" stroke="#00FF19" />
+                                </LineChart>
+                            </div>
+                        </div>
+                        <div className="pie-chart">
+                            <div className="overlap">
+                                <div className="div-3" />
+                                <div className="status">
+                                    {Object.entries(dashBoardInfo?.licenseCountMap || {}).map(([license, count]) => {
+
+                                        const cleanLicenseName = license.replace(/[^a-zA-Z]/g, '');
+                                        if ( cleanLicenseName == 'MIT' || cleanLicenseName == 'Apache' || cleanLicenseName == 'GPLorlater' || cleanLicenseName == 'GPLonly' || cleanLicenseName == 'OpenSSL' )
+                                            return (
+                                                <div className={cleanLicenseName} key={license}>
+                                                    <div className="text-wrapper-2">{count}</div>
+                                                    <div className={`text-wrapper-${cleanLicenseName}`}>{license}</div>
+                                                    <div className={`${cleanLicenseName}-color`}></div>
+                                                </div>
+                                            );
+                                        else return null;
+                                    })}
```

Comment:
When the `spdx id` is similar, such as `GPL-2.0-only` and `GPL-3.0-only`, they belong to the same div class, `GPLonly`. As a result, they overlap each other.
![image](https://github.com/Samsung/LPVS/assets/5847128/1c8ccd75-8f00-470d-abb7-0d86be1afa56)

They should be separated from each other and be calculated relatively instead of absolute position.

Reply:
![image](https://github.com/Samsung/LPVS/assets/57388014/220964f3-5cdd-41e9-98a7-39058cc934ad)

I've revised it according to the review you left.
I'd appreciate it if you could check if it's been modified properly.